OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -62,48 +67,95 @@ public Turreted(ActorInitializer init, TurretedInfo info)
 		{
 			this.info = info;
 			TurretFacing = GetInitialTurretFacing(init, info.InitialFacing);
+
+			if (info.RandomRealign)
+				randomFacing = init.Self.World.SharedRandom.Next(0, 256);
```

Comment:
Yeah, unfortunately @Unit158 is right here. I tried doing it with `INotifyBecomingIdle` today, it won't work :/